### PR TITLE
Update CI to test provider against the 3 different editions of SonarQube

### PR DESCRIPTION
**Pull request by felixlut**
This PR is in draft mode until #143 is merged. Once it is, I will rebase this PR on top of that, and re-write this PR description.

### CODE REVIEW

**Comment by jdamata** [comment]
Do you think it doesn't make sense to have tests running against latest images as well as the current LTS version?

File: .github/workflows/acceptance-tests.yml [hunk 1]
```diff
@@ -32,10 +36,10 @@ jobs:
           SONAR_USER: admin
           SONAR_PASS: admin
       - name: Run code coverage
-        if: ${{ github.ref == 'refs/heads/master' }}
+        if: ${{ github.ref == 'refs/heads/master' && matrix.image_tag == 'developer'}}
```
Comment (jdamata) [inline .github/workflows/acceptance-tests.yml:39]: lets make this enterprise, since all tests will run against enterprise

**Review by jdamata — APPROVED** [review]

**Comment by felixlut** [comment]
> Did you think it doesn't make sense to have tests running against latest images as well as the current LTS version? Curious why you removed it

The LTS and community tags are pointing to the same image:
https://hub.docker.com/_/sonarqube

**Comment by felixlut** [comment]
I could change the images to {edition}-lts if that is clearer. 

Also, i have no idea what the 2 datacenter editions are so i skipped running tests against them. Do you know what they are, and if it would be a good idea to add them to the CI?

**Comment by jdamata** [comment]
> I could change the images to `lts-{edition}` if that is clearer.
> 
> Also, i have no idea what the 2 datacenter editions are so i skipped running tests against them. Do you know what they are, and if it would be a good idea to add them to the CI?

had no idea latest and lts were the same. i think we are fine as is

datacenter is similar to enterprise and probably the same on the api side of things. Datacenter focuses on scalability and resiliency. I dont think we need it here. 

Thanks for your work on this

**Pull request merged**